pkgs/development/compilers/dotnet/default.nix

```diff
@@ -8,9 +8,16 @@ https://dotnet.microsoft.com/download/dotnet
 { callPackage, icu70, icu }:
 let
   buildDotnet = attrs: callPackage (import ./build-dotnet.nix attrs) {};
-  buildAspNetCore = attrs: buildDotnet (attrs // { type = "aspnetcore"; });
-  buildNetRuntime = attrs: buildDotnet (attrs // { type = "runtime"; });
-  buildNetSdk = attrs: buildDotnet (attrs // { type = "sdk"; });
+  buildAttrs = {
+    buildAspNetCore = attrs: buildDotnet (attrs // { type = "aspnetcore"; });
+    buildNetRuntime = attrs: buildDotnet (attrs // { type = "runtime"; });
+    buildNetSdk = attrs: buildDotnet (attrs // { type = "sdk"; });
+  };
+
+  ## Files in versions/ are generated automatically by print-hashes.sh ##
```

Comment:
I think `print-hashes.sh` should be renamed to something along the lines of `update.sh`, now that its completely automatic. It doesn't just print the hashes anymore.

Reply:
Comment still needs to be updated